codeexamples/SYCLUsingCUDAStaticLibraryOfKernels/StaticLibrarySYCLUsesCUDAKernels/.vscode/settings.json

```diff
@@ -0,0 +1,84 @@
+{
+    "programName": "DXTC",
+    "files.associations": {
```

Comment:
The `files.associations` setting is irrelevant to the project, please remove

Reply:
These have been removed.

Reply:
The file seems to be still in the PR in the latest commit, it's just completely empty now. Can it be removed?

Reply:
I cannot see a way to delete a file on the upstream repository. I need the file locally, so it cannot be deleted there. VSCode will 'edit' file (because it is its set of 4 required .cpp project files - they always get formed) when the user interacts with the project.      If I had ignored the file from the start this would not be a problem.  I way needs to be found to remove the file. Empty file is the next best thing as it will be filled out by VSCode.